### PR TITLE
Test/changes

### PR DESCRIPTION
**Pull request by EricSmekens**
Hi,

I wanted to do have some extra practice in writing some tests.
Added some sanity-basic tests for checking if the hideCollidedLabels is working.

Feedback is welcome ofcourse!

#hacktoberfest

### CODE REVIEW

File: test/dataLabel/dataLabelManagerTest.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ module powerbi.extensibility.utils.chart.dataLabel.test {
     import RectOrientation = powerbi.extensibility.utils.chart.dataLabel.RectOrientation;
     import ContentPositions = powerbi.extensibility.utils.chart.dataLabel.ContentPositions;
     import OutsidePlacement = powerbi.extensibility.utils.chart.dataLabel.OutsidePlacement;
+    import LabelPosition = powerbi.extensibility.utils.chart.dataLabel.LabelPosition;
```
Comment (EugeneElkin) [inline test/dataLabel/dataLabelManagerTest.ts]: Why is there an import that is not used anywhere?

File: test/dataLabel/dataLabelManagerTest.ts [hunk 2]
```diff
@@ -61,10 +62,19 @@ module powerbi.extensibility.utils.chart.dataLabel.test {
             });
 
             it("Get Label info - all values Provided", () => {
-                let result: IDataLabelInfo = labelManager.getLabelInfo({ maximumMovingDistance: 12 });
+                let result: IDataLabelInfo = labelManager.getLabelInfo({ maximumMovingDistance: 16 });
```
Comment (EugeneElkin) [inline test/dataLabel/dataLabelManagerTest.ts]: Why did you change the value in the existent test?

Comment (EricSmekens) [reply test/dataLabel/dataLabelManagerTest.ts]: Changed it back.

File: test/dataLabel/dataLabelManagerTest.ts (hunk 1 above)
Comment (EricSmekens) [reply test/dataLabel/dataLabelManagerTest.ts]: Changed the way of importing.

This is the only test in a module, and since the namespace conflict with `powerbi`, I had to change the module name.
